docs/guide.installation.rst

```diff
@@ -99,22 +153,22 @@ and thus have different requirements.
 
 Select scripts
 ---------------
-
-To configure the environment of your agent, autopilot can run a set of scripts (see :func:`.setup_autopilot.run_script` and :func:`.setup_autopilot.list_scripts`)
+Now you will see a menu of potential scripts that can be run. Select the scripts you want to run, and then hit "OK". Note that even the simplest task ("free water") requires pigpio, so you may want to include that one. You can see the commands that will be run in each of these scripts in :func:`.setup_autopilot.run_script` and :func:`.setup_autopilot.list_scripts`.
```

Comment:
will also add a comment that we use a modified version of pigpio that lets us get absolute timestamps from pigpiod, increases the number of scripts, etc. (the quiet goal is to eventually merge the core ops of pigpio into autopilot which would streamline a lot and give us more control over low-level gpio control)